bedrock/firefox/templates/firefox/firstrun/ravioli.html

```diff
@@ -0,0 +1,74 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block gtm_page_id %}data-gtm-page-id="/firefox/firstrun/"{% endblock %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{_('Choose the independent browser')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_firstrun_ravioli' %}
+{% endblock %}
+
+{% block page_title_full %}
+  {{ _('Search and see, with Yahoo Search.') }}
+{% endblock %}
+
+{% block site_header %}
+<header id="masthead">
+  <div id="tabzilla">
+    <a href="{{ url('mozorg.home') }}" data-link-type="nav" data-link-name="tabzilla">Mozilla</a>
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="container">
+  <h2>
+    <a href="{{ url('firefox.new') }}">
+      {{ high_res_img('firefox/firstrun/learnmore/yahoo/firefox-logo.png', {'alt': 'Firefox', 'width': '97', 'height': '100'}) }}
+    </a>
+  </h2>
+
+  <h1>{{ _('Search and see, with Yahoo Search.') }}</h1>
+
+  <p>{{ _('Yahoo Search brings you relevant results with high quality images and video for a more visual search experience.') }}</p>
+
+  <figure class="ravioli">
+    {{ high_res_img('firefox/firstrun/ravioli.png', {'alt': 'Yahoo Search'}) }}
+    <figcaption>
+      {{ _('Yahoo is one of Mozilla’s search partners. <a rel="external" href="%s">Learn more.</a>')|format('https://blog.mozilla.org/blog/2015/11/11/a-new-yahoo-search-experience-for-firefox-users-in-u-s/') }}
```

Comment:
Split this into two strings? Then "Learn more." can be its own string without the HTML. I know this is English only for now but might this eventually be localized?